### PR TITLE
Add "Reading" case

### PR DESCRIPTION
**Pull request by Richard-Cao**
Add "Reading" case

### CODE REVIEW

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -622,6 +622,15 @@ var apps = [
     linkAppStore: 'https://itunes.apple.com/cn/app/hong-bei-bang-hai-liang-hong/id1007812319?mt=8',
     author: 'Hongbeibang'
   },
+  {
+    name: 'Reading',
+    icon: 'http://7xr0xq.com1.z0.glb.clouddn.com/about_logo.png',
+    linkPlayStore: 'http://www.wandoujia.com/apps/com.reading',
```
Comment (chirag04) [inline website/src/react-native/showcase.js]: use link key here

Comment (bestander) [inline website/src/react-native/showcase.js]: @chirag04 you wanted to add something here?

Comment (chirag04) [inline website/src/react-native/showcase.js]: he needs to change `linkPlayStore` key to be `link` key.

`linkPlayStore` is used to differentiate `ios` and `android` links.

**Comment by Richard-Cao** [comment]
update

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**